Dockerfile

```diff
@@ -67,9 +67,9 @@ COPY ./scripts/inject-dynamic-env.sh /docker-entrypoint.d/00-inject-dynamic-env.
 
 RUN chmod +x /docker-entrypoint.d/00-inject-dynamic-env.sh
 
-# Install bash and update vulnerable packages
+# Update vulnerable packages
 RUN apk update && \
-    apk add --no-cache bash
+    apk upgrade
```

Comment:
`apk upgrade` or any other package managed upgrades are not allowed. The reason is that these official base images are scanned by Docker Hub and the results are published afterwards. Changing the image this way would mean those scan results are no longer valid. If you have any issues with the current image, consider upgrading to a newer tag.